ts/Stock/StockToolsGui.ts

```diff
@@ -1223,6 +1227,34 @@ class Toolbar {
             }
         });
     }
+    /**
+     * Check if any of the price indicators are enabled.
+     * @private
+     * @function bindingsUtils.isLastPriceEnabled
+     *
+     * @param {array} series
+     *        Array of series.
+     *
+     * @return {string}
+     *         Tells which indicator is enabled.
+     */
+    public isPriceIndicatorEnabled = function (series: SeriesOptions[]): boolean {
+        let priceIndicatorEnabled = false;
+
+        series.forEach(function (serie): void {
+            const lastVisiblePrice = serie.lastVisiblePrice && serie.lastVisiblePrice.enabled,
+                lastPrice = serie.lastPrice && serie.lastPrice.enabled;
+
+            if (lastVisiblePrice || lastPrice) {
+                priceIndicatorEnabled = true;
+            }
+
+            if (!lastVisiblePrice && !lastPrice) {
+                priceIndicatorEnabled = false;
+            }
+        });
```

Comment:
This `forEach` in fact is testing only the last series. No matter what we decide for the first series, the last one can define `priceIndicatorEnabled` according to its own internal options.

For example, the first series has `lastPrice` enabled and it's setting `priceIndicatorEnabled` to true. Then the last series has both options disabled, resulting in `priceIndicatorEnabled = false`. If we reverse the series order, then we will always receive `priceIndicatorEnabled=true`, no matter what was set in the first series.